README.md

```diff
@@ -77,13 +80,14 @@ The module is installed in A/Cs and humidifiers that are either manufactured or
    curl -ik 'http://localhost:8888/hisense/command?property=t_power&value=ON'
    ```
 ## Run as a service
+Assuming your username is "pi"
 
 1. Create a dedicated directory for the script files, and move the files to it.
    Pass the ownership to root. e.g.:
    ```bash
    sudo mkdir /usr/lib/hisense
-   sudo mv hisense.py config.json /usr/lib/hisense
-   sudo chown root:root /usr/lib/hisense/*
```

Comment:
Back at that version it was the way it should be configured. Current implementation on branch feature/hass_with_api uses it differently, so it will change